docs/udales-namoptions-overview.md

```diff
@@ -228,22 +204,44 @@ BCs at the bottom (BCbot; only effective if not covered with ground facets): 1 =
 | nfctsecs_v | 0 | `INTEGER` | Number of facet sections on v-grid. | - |
 | nfctsecs_w | 0 | `INTEGER` | Number of facet sections on w-grid. | - |
 | nfctsecs_c | 0 | `INTEGER` | Number of facet sections on c-grid. | - |
-| lnorec | .false. | .true., .false. | Switch for not using reconstruction. | - |
+| lnorec | .false. | .true. or .false. | Switch for not using reconstruction. | - |
+
+
+## Namelist ENERGYBALANCE
+
+| Name | Default | Possible values | Description | Unit |
+| ---- | ------- | --------------- | ----------- | ---- |
+| lEB | .false. | .true. or .false. | Switch for using the facet energy balance. | - |
+| lwriteEBfiles | .false. | .true., .false. | Switch for writing facet temperatures and energy budget to file. | - |
+| lconstW | .false. | .true. or .false. | Switch whether soil moisture is assumed as constant in time (.true.) or the evaporated water is from the soil (.false.). | - |
+| dtEB | 10. | `REAL` | Time interval between calculations of facet energy balance. | s |
+| bldT | 0. | `REAL` | Internal temperature of the buildings. | [K] |
+| flrT | 0. | `REAL` | Internal temperature of the ground. | [K] |
+| wsoil | 0. | `REAL` | Water content of soil. | [kg/m3] |
+| wgrmax | 450. | `REAL` | Maximum water content. | [kg/m3] |
+| wwilt | 171. | `REAL` | Water content at wilting point. | [kg/m3] |
+| wfc | 313. | `REAL` | Water content at field capacity. | [kg/m3] |
+| skyLW | 0. | `REAL` | Long-wave radiation from the sky. | - |
+| GRLAI | 2. | `REAL` | Leaf area index of a green roof. | - |
+| rsmin | 110. | `REAL` | Minimum resistance of soil/plant. | - |
+| nfaclyrs | 3 | `INTEGER` | Number of layers making up each facet (nwalllayers in uDALES v1). | - |
+| lvfsparse | .false. | .true. or .false. | Switch for view factors in sparse (text) format. | - |
+| nnz | 0 | `INTEGER` | Number of non-zero view factors (only used with sparse view factor format. | - |
+| lperiodicEBcorr | .false. | .true. or .false. | Switch for preventing over-heating and moisture saturation in periodic simualtions. | - |
+| sinkbase | 0 | `INTEGER` > 0  | k index above which the periodicEBcorr sink is applied (should be above height of tallest building). | - |
+| sinkbase | 0 | `INTEGER` > 0  | k index above which the periodicEBcorr sink is applied (should be above height of tallest building). | - |
+| fraction | 0 | `Real` > 0  | Ratio of domain height to uncapped boundary layer height. | - |
 
 ## Namelist SCALARS
 
 | Name | Default | Possible values | Description | Unit |
 | ---- | ------- | --------------- | ----------- | ---- |
-| nsv | 0 | | See [DALES](https://github.com/dalesteam/dales/blob/master/utils/doc/input/Namoptions.pdf). | |
+| nsv | 0 | `INTEGER` > 0 | Number of passive scalars. | - |
 | lreadscal | .false. | .true., .false. | Switch for reading scalar pollutant field (warm start). *Deprecated, will be removed in the future.* | - |
```

Comment:
Is it deprecated now @madDipanjan?

Reply:
xS, yS, zS, SS, SigS these variables have been modified now and moved to &INPS. uDALES reads scalarsource files generated through write_inputs.